src/models/reposyncstate.ts

```diff
@@ -208,6 +208,18 @@ export class RepoSyncState extends Model {
 		}));
 	}
 
+	static async findRepositoriesBySubscriptionIdAndRepoName(subscriptionId: number, repoName: string): Promise<RepoSyncState[] | null> {
+		return RepoSyncState.findAll({
+			where: {
+				subscriptionId,
+				repoName: {
+					[Op.iLike]: `%${repoName}%`
```

Comment:
See the comment I left in this PR explaining why we need to do partial matching https://github.com/atlassian/github-for-jira/pull/2149/files#r1212443079